woocommerce_fusion/tasks/sync_sales_orders.py

```
@@ -25,21 +27,33 @@
+			exc_info=True,
+		)
 		return dict(WC_ORDER_STATUS_MAPPING), dict(WC_ORDER_STATUS_MAPPING_REVERSE)
 
 
 def _get_allowed_inbound_statuses(woocommerce_server: str) -> list[str]:
 	"""Use server method to get allowed inbound statuses with default fallback."""
+	logger = get_logger()
 	try:
 		wc_server = frappe.get_cached_doc("WooCommerce Server", woocommerce_server)
 		return wc_server.get_allowed_inbound_statuses()
 	except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)